apps/appgen/appgen.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+echo "TizenRT Application Generator"
+echo "======================= v 1.0"
+
+read -p "Enter application name: " APP_NAME
+
+APP_NAME_UPPER=${APP_NAME^^}
+APP_NAME_UPPER="${APP_NAME_UPPER// /_}"
+APP_NAME_LOWER=${APP_NAME,,}
+APP_NAME_LOWER="${APP_NAME_LOWER// /_}"
+ENTRY_FUNC="${APP_NAME_LOWER}_main"
+
+echo "[Summary]"
+echo "Application Name: ${APP_NAME}"
+echo "Configuration Key: CONFIG_APP_${APP_NAME_UPPER}"
+echo "Entry Function: ${ENTRY_FUNC}"
+
+read -p "Continue? (y/N): " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || exit 1
+
+echo "Generating..."
+
+KCONFIG_FILENAME="examples/${APP_NAME_LOWER}/Kconfig"
+KCONFIG_ENTRY_FILENAME="examples/${APP_NAME_LOWER}/Kconfig_ENTRY"
+MAIN_FILENAME="examples/${APP_NAME_LOWER}/${ENTRY_FUNC}.c"
+MAKE_DEFS_FILENAME="examples/${APP_NAME_LOWER}/Make.defs"
+MAKEFILE_FILENAME="examples/${APP_NAME_LOWER}/Makefile"
+
+mkdir "examples/${APP_NAME_LOWER}"
```

Comment:
Is this path correct? appgen folder is not in the same path with examples folder

Reply:
As the README.md, currently, user has to run this shell script at the only 'apps' directory. But we can improve this can run at the other directories.